packages/contentstack-import/src/import/modules/entries.ts

```diff
@@ -817,19 +823,17 @@ export default class EntriesImport extends BaseClass {
   async removeAutoCreatedEntries(): Promise<void> {
     const onSuccess = ({ response, apiData: { entryUid } }: any) => {
       // NOTE Remove entry from list
-      remove(this.entriesForVariant, {
-        entry_uid: entryUid,
-        locale: this.importConfig?.master_locale?.code,
-      });
+      this.entriesForVariant = this.entriesForVariant.filter(
```

Comment:
Why was this code updated to inline logic instead of using a method? This change has introduced redundancy across the codebase

Reply:
Since this was an existing implementation, I just resolved the type issues in all places.